README.md

```diff
@@ -1,23 +1,16 @@
 Chitter Challenge
```

Comment:
Perhaps could flesh out the Readme some more with screenshots, and 'how to use' descriptions etc.